src/main/resources/public/config-check.js

```diff
@@ -12,39 +11,33 @@ $(function() {
         if (response.TWILIO_ACCOUNT_SID && response.TWILIO_ACCOUNT_SID != '' &&
             response.TWILIO_API_KEY && response.TWILIO_API_KEY != '' && response.TWILIO_API_SECRET) {
 
-            if (response.TWILIO_CONFIGURATION_SID && response.TWILIO_CONFIGURATION_SID != '') {
-                $('#videoDemoButton').addClass('btn-success');
-            } else {
-                $('#videoDemoButton').addClass('btn-danger');  
-            }
-
             if (response.TWILIO_CHAT_SERVICE_SID && response.TWILIO_CHAT_SERVICE_SID != '') {
```

Comment:
For CSS/HTML sanity reasons I would add `$('#videoDemoButton').addClass('btn-success');` and remove the `btn-success` class from the HTML. So that when we add `btn-danger` we don't have both classes applied.
Ref: remove `btn-success` from https://github.com/TwilioDevEd/sdk-starter-java/pull/16/files#diff-99ff2f55677b1063bbd730b6718e6f13R49